src/test/scala/sbtwhitesource/BaseActionSpec.scala

```diff
@@ -11,15 +11,41 @@ class BaseActionSpec extends WordSpec with Matchers {
   "The base action" should {
     "merge standard and native jars of the same artifacts" in {
       val nativeUrl = new URL("https://repo1.maven.org/maven2/com/github/jnr/jffi/1.2.16/jffi-1.2.16-native.jar")
-      val nativeArtifact: Artifact = Artifact("com.github", "jar", "jar", Some("native"), Vector(), Some(nativeUrl))
+      val nativeArtifact: Artifact = Artifact("jffi", "jar", "jar", Some("native"), Vector(), Some(nativeUrl))
```

Comment:
This doesn't really matter, but I'm pretty sure that this is meant to be the artifact ID rather than group ID/organization. I changed it for consistency and clarity.